pkg/instancetype/instancetype.go

```diff
@@ -82,6 +81,25 @@ func GetPreferredTopology(preferenceSpec *instancetypev1beta1.VirtualMachinePref
 	return preferredTopology
 }
 
+const defaultSpreadRatio uint32 = 2
+
+func GetSpreadOptions(preferenceSpec *instancetypev1beta1.VirtualMachinePreferenceSpec) (uint32, instancetypev1beta1.SpreadAcross) {
+	ratio := defaultSpreadRatio
+	if preferenceSpec.PreferSpreadSocketToCoreRatio != 0 {
+		ratio = preferenceSpec.PreferSpreadSocketToCoreRatio
```

Comment:
This is for backwards compatibility, right?

Reply:
Yes.

Reply:
ACK